Source/WebCore/loader/FrameLoadRequest.h

```diff
@@ -111,6 +112,8 @@ class FrameLoadRequest {
     void setAdvancedPrivacyProtections(OptionSet<AdvancedPrivacyProtections> policy) { m_advancedPrivacyProtections = policy; }
     OptionSet<AdvancedPrivacyProtections> advancedPrivacyProtections() const { return m_advancedPrivacyProtections; }
 
+    NavigationHistoryBehavior navigationHistoryBehavior() const { return m_navigationHistoryBehavior; }
+    void setNavigationHistoryHandling(NavigationHistoryBehavior historyHandling) { m_navigationHistoryBehavior = historyHandling; }
```

Comment:
Can the getter and setter for this variable have the same name? Instead of `HistoryHandling` and `HistoryBehavior`.

Reply:
Sure. The spec happens to call the type HistoryBehavior and the variable always historyHandling.

I guess I'll go with HistoryHandling.